src/ru/skypro/Main.java

```diff
@@ -4,5 +4,20 @@
 
     public static void main(String[] args) {
 	// write your code here
+        String firstName = "Ivan";
+        String middleName = "Ivanovich";
+        String lastName = "Ivanov";
+        String fullName = "Ivanov Ivan Ivanovich";
```

Comment:
здесь стоило выполнить конкатенацию значений предыдущих переменных, чтобы составить данное поле
```suggestion
        String fullName = firstName + " " + middleName + " " + lastName;
```